apps/frontend/src/libs/components/search-input/search-input.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+	type Control,
+	type FieldErrors,
+	type FieldPath,
+	type FieldValues,
+} from "react-hook-form";
+
+import { Input } from "~/libs/components/components.js";
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+import { useAppSearch } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type Properties<T extends FieldValues> = {
+	className?: string;
+	control: Control<T, null>;
+	errors: FieldErrors<T>;
+	hasVisuallyHiddenLabel?: boolean;
+	label: string;
+	name: FieldPath<T>;
+	onSearch?: (value: string) => void;
+	placeholder?: string;
+};
+
+const SearchInput = <T extends FieldValues>({
+	className = "",
+	control,
+	errors,
+	hasVisuallyHiddenLabel = true,
```

Comment:
```suggestion
	hasVisuallyHiddenLabel = false,
```
Default value should be false